HW__2.postman_collection.json

```diff
@@ -0,0 +1,238 @@
+							"        console.log(typeof(e.name) === \"string\" ? \"name's type = string - true\":\"name's type = string - false\");\r",
+							"        pm.test(\"Status code is 200\", function () {\r",
+							"            pm.response.to.have.status(200);\r",
+							"        });\r",
+							"        }\r",
+							"})\r",
+							"\r",
+							"\r",
+							"   "
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "https://rickandmortyapi.com/api/character",
+					"protocol": "https",
+					"host": [
+						"rickandmortyapi",
+						"com"
+					],
+					"path": [
+						"api",
+						"character"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "HW_2.1",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"//const names1 = ['Anastasia', 'Svetlana', 'Nikita', 'Viktar', 'Egor'];\r",
+							"//const names2 = ['Svetlana', 'Egor', 'Nikita'];\r",
+							"function excluding_names(names1,names2){\r",
+							"\r",
+							"    console.log(_.difference(names1,names2))}\r",
+							"\r",
+							"//excluding_names([1,2,3,4],[2,56,2,6,7])"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "httpbin.org/get",
+					"host": [
+						"httpbin",
+						"org"
+					],
+					"path": [
+						"get"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "HW_2.2",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"const ajv = require('ajv');\r",
+							"const moment = require(\"moment\");\r",
+							"const uid = require(\"uuid\");\r",
+							"//const jsonBody = pm.response.json();\r",
+							"const ajvInstance = new ajv({logger: console});\r",
+							"const schema = {\r",
+							"    //\"type\": \"object\",\r",
+							"    //\"properties\": {\r",
+							"            \"date\": \"string\",\r",
+							"            \"guid\": \"string\",\r",
+							"            \"randomString\": \"string\",\r",
+							"\r",
+							"}//}\r",
+							"//console.log(typeof(moment().format('MMMM Do YYYY')));\r",
+							"//console.log({{$randomFirstName}});\r",
+							"const data1 = {\r",
+							"\"date\": moment().format('MMMM Do YYYY'),\r",
+							"\"guid\": uid(),\r",
+							"\"randomString\": moment() + uid(),\r",
+							"}\r",
+							"pm.test('valid', () => {\r",
+							"    pm.expect(ajvInstance.validate(schema, data1)).to.be.true;\r",
+							"});\r",
+							"//console.log(data1);\r",
+							"console.log(ajvInstance.errors);\r",
```

Comment:
We could place console.log(ajvInstance.errors) to the if statement:

```
if (ajvInstance.errors) {
    console.log("Current schema issues:");
    console.log(ajvInstance.errors);
}
```
In this case, we console.log() if our schema has some issues. Otherwise we're posting null to the console.